src/main/java/me/coley/recaf/decompile/cfr/CfrDecompiler.java

```diff
@@ -87,8 +92,13 @@ private String clean(String decompilation, String name) {
 		// Fix inner class names being busted, "Outer.1" instead of "Outer$1"
 		String simple = name.contains("/") ? name.substring(name.lastIndexOf('/') + 1) : name;
 		if(simple.contains("$")) {
-			// They have "." instead of "$"
-			decompilation = decompilation.replace(simple.replace("$", "."), simple);
+			// They will replace last of "$" with "."
+			decompilation = decompilation.replace(
+					new StringBuilder(simple).replace(
+							simple.lastIndexOf("$"), simple.lastIndexOf("$") + 1, "."
+					).toString(),
```

Comment:
Is there a reason why this needs to be wrapped in a `StringBuilder`?

Reply:
Only need to replace the last "$" with "."